packages/augur-ui/src/modules/swap/components/swap-row.styles.less

```diff
@@ -51,11 +51,11 @@
         }
 
         svg:last-of-type {
-          fill: var(--color-outcome-seven);
           width: @size-12;
 
           > g {
             stroke-width: 2;
+            fill: var(--color-outcome-seven);
```

Comment:
we really should look at this, as outcome seven is completely different colors based on the theme. if this is always supposed to be a specific color, we should just make an exception -- see `--color-facebook-1` kinda thing. we have hardcoded third party colors we need due to branding so we have an exception where it's the same no matter the theme